knimin/lib/tests/test_data_access.py

```diff
@@ -60,6 +60,16 @@ def test_get_unconsented(self):
         exp = ['000014660', datetime.date(2015, 3, 25), 'REMOVED']
         self.assertEqual(obs[0], exp)
 
+    def test_search_kits(self):
```

Comment:
why was a test for `search_kits` added when it doesn't look like the code was modified? Glad to have more tests, but would have assumed they would be for the modified methods